app/searchhandler.py

```diff
@@ -287,10 +310,17 @@ async def get(self, request):
             search_criteria_with_range['rangeHigh'] = high_value
             search_criteria_with_range['rangeLow'] = low_value
 
-            retrieve_employee_info = get_employee_records(
-                search_criteria_with_range)
+            if previous_user_missing_device != False:
+                # if the checkbox is not false, the default value
+                retrieve_emplyee_info = get_employee_records_no_device( 
+                                        search_criteria_with_range )
+            else:
+                # Works - riade exception code to prove it
```

Comment:
An exception doesn't actually seem to be raised here - was this a debugging line?

Reply:
Ahh yes, originally there was an exception there but was removed, must've missed that line too!